packages/core/scripts/copy-build-info.js

```diff
@@ -47,10 +47,48 @@ assert(
     readJSON('artifacts/@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol/ProxyAdmin.dbg.json').buildInfo,
 );
 
-const buildInfo = readJSON(jsonRelativePath);
-const reducedInfo = { solcLongVersion: buildInfo.solcLongVersion, input: buildInfo.input };
+let buildInfo = readJSON(jsonRelativePath);
 
-const sources = reducedInfo.input.sources;
+// Keep only relevant sections of output.contracts
+const contractFiles = buildInfo.output.contracts;
+for (const contractFile in contractFiles) {
+  const contractNames = contractFiles[contractFile];
+  for (const contractName in contractNames) {
+    contractNames[contractName] = {
+      abi: contractNames[contractName].abi,
+      evm: contractNames[contractName].evm,
+      metadata: contractNames[contractName].metadata, // metadata is needed to determine the license type
+    };
+  }
+}
+
+// Adjust correponding output selection
+const input = buildInfo.input;
+
+const origSelection = input.settings.outputSelection['*']['*'];
+assert(origSelection.includes('abi'));
+assert(origSelection.includes('evm.bytecode'));
+assert(origSelection.includes('evm.deployedBytecode'));
+assert(origSelection.includes('evm.methodIdentifiers'));
+assert(origSelection.includes('metadata'));
+
+input.settings.outputSelection['*'] = {
+  '*': ['abi', 'evm.bytecode', 'evm.deployedBytecode', 'evm.methodIdentifiers', 'metadata'],
+};
+
+// Keep only relevant sections of build info
```

Comment:
What do we intend to remove? Can we specify that in the comment?

If I understand correctly we're removing only from the `output` object and keeping only `output.contracts` as filtered above, but also removing the AST for example.

I assume this is to reduce the package size? We could also specify that in a comment somewhere.